doc/library/config.txt

```diff
@@ -102,22 +102,18 @@ import theano and print the config variable, as in:
 
 .. attribute:: device
 
-    String value: either ``'cpu'``, ``'gpu'``, ``'gpu0'``, ``'gpu1'``,
-    ``'gpu2'``, or ``'gpu3'``
+    String value: either ``'cpu'``, ``'cuda'``, ``'cuda0'``, ``'cuda1'``,
+    ``'opencl0:0'``, or ``'opencl0:1'`` ...
```

Comment:
Should we still mention that gpu / gpuX works and maps to the old, deprecated back-end?